### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by mikouaj**
* Introduced User Guide
* Updated `README.md` so it shows basic usage and refers to the user guide

### CODE REVIEW

File: docs/user-guide.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+  --git-policy-dir "gke-policies"
+  ```
+
+*NOTE*: currently the tool does not support authentication for GIT policy repositories.
+
+### Specifying local policy source
+
+The local policy source directory can be specified with a command line flags or in a [configuration file](#configuration-file).
+
+* `local-policy-dir` is a local policy directory to search for policy files
```
Comment (ewojtach) [inline docs/user-guide.md:198]: path to local policy directory?

File: docs/user-guide.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+silent: true
+clusters:
+  - name: prod-central
+    project: my-project-one
+    location: europe-central2
+  - id: projects/my-project-two/locations/europe-west2/clusters/prod-west
+policies:
+  - repository: https://github.com/google/gke-policy-automation
+    branch: main
+    directory: gke-policiese
```
Comment (ewojtach) [inline docs/user-guide.md:253]: gke-policies

**Review by ewojtach — APPROVED** [review]
great documentation!

File: docs/user-guide.md (hunk 1 above)
Comment (mikouaj) [reply docs/user-guide.md:198]: Good point, thanks :-)

File: docs/user-guide.md (hunk 2 above)
Comment (mikouaj) [reply docs/user-guide.md]: Fixed

**Pull request merged**